examples/website/src/shared/utility.ts

```diff
@@ -125,7 +125,12 @@ export function transformCssStringToObject(styleCustom: string): object {
   return styleRules.reduce((previousValue: object, currentValue: string) => {
     const [key, value] = currentValue.split(':')
     if (key && value) {
-      return Object.assign(previousValue, {[key.trim()]: value.trim()})
+      return Object.assign(previousValue, {
```

Comment:
what function! don't get, why the heck it is necessary to write such functions. but if it works still for other iframes, it's ok for me.

Reply:
Yes was necessary to handle properties from the iframe